test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -102,4 +112,131 @@ describe('IexecAccessors', async () => {
             '0xfc2178d8b8300e657cb9f8b5a4d1957174cf1392e294f3575b82a9cea1da1c4b',
         );
     });
+
+    describe('Task Result Tests', () => {
+        let iexecWrapper: IexecWrapper;
+        let [appAddress, datasetAddress, workerpoolAddress]: string[] = [];
+        let [requester, scheduler, worker1]: SignerWithAddress[] = [];
+        let ordersAssets: OrdersAssets;
+        const { results, resultDigest } = buildUtf8ResultAndDigest('result');
+
+        beforeEach(async () => {
+            let accounts = await getIexecAccounts();
+            iexecWrapper = new IexecWrapper(proxyAddress, accounts);
+            ({ requester, worker1, scheduler } = accounts);
+            ({ appAddress, datasetAddress, workerpoolAddress } = await iexecWrapper.createAssets());
+            ordersAssets = {
+                app: appAddress,
+                dataset: datasetAddress,
+                workerpool: workerpoolAddress,
+            };
+        });
+        it('Should get result of task', async function () {
+            const orders = buildOrders({ assets: ordersAssets, requester: requester.address });
+            const { dealId, taskId, taskIndex } = await iexecWrapper.signAndMatchOrders(
+                ...orders.toArray(),
+            );
+
+            await iexecPocoAsAnyone.initialize(dealId, taskIndex).then((tx) => tx.wait());
+
+            const workerTaskStake = await iexecPocoAsAnyone
+                .viewDeal(dealId)
+                .then((deal) => deal.workerStake.toNumber());
+            const { resultHash, resultSeal } = buildResultHashAndResultSeal(
+                taskId,
+                resultDigest,
+                worker1,
+            );
+            const schedulerSignature = await buildAndSignContributionAuthorizationMessage(
+                worker1.address,
+                taskId,
+                AddressZero,
+                scheduler,
+            );
+            await iexecWrapper.depositInIexecAccount(worker1, workerTaskStake);
+            await iexecPocoAsAnyone
+                .connect(worker1)
+                .contribute(taskId, resultHash, resultSeal, AddressZero, '0x', schedulerSignature)
+                .then((tx) => tx.wait());
+
+            await iexecPocoAsAnyone
+                .connect(worker1)
+                .reveal(taskId, resultDigest)
+                .then((tx) => tx.wait());
+            await iexecPocoAsAnyone
+                .connect(scheduler)
+                .finalize(taskId, results, '0x')
+                .then((tx) => tx.wait());
+            const task = await iexecPocoAsAnyone.viewTask(taskId);
+            await expect(task.status).to.equal(TaskStatusEnum.COMPLETED);
+            await expect(await iexecPocoAsAnyone.callStatic.resultFor(taskId)).to.equal('0x');
+        });
```

Comment:
I think the test with result callback is missing

Reply:
Done

Reply:
Add `callback` in `buildOrders(..)`?